Boeing viz/scrollytelling

dlousa92 opened the pull request:
Here's the initial pull request. I created a new branch "booeing-viz/scrollytelling" and I'm working off that. I added the new svg that removed all that whitespace. I also added opacity to the background of the paragraphs (Are you honed in on making those backgrounds white? Was playing around with transparent gray and it looked nice on my end) . I Had to play around with margin values a little bit and the timing of the waypoint triggers so that the paragraphs look nice. 

Also there are a lot of commits that Anne made previously here.

jnschrag commented on assets/_sass/custom-viz/boeing/_scrollytelling.scss:
```diff
@@ -0,0 +1,29 @@
+=        Scrollytelling        =
+==============================*/
+
+.label-group {
```
For the `.label-group`, `.hidden`, and `.display-none` classes, I would prefix them or otherwise increase the specificity to ensure they don't impact anything outside of the interactive. For example, you could do this:
```scss
#boeignScroll {
  .js-hidden {}
  .js-display-none {}
  .label-group {}
}
```

Note the `js-` prefix, which is used to indicate that those classes are toggled on/off via JavaScript

jnschrag commented on assets/_sass/custom-viz/boeing/_scrollytelling.scss:
```diff
@@ -0,0 +1,29 @@
+}
+
+.boeing-js__opacity-not-viewed {
+  opacity: .5;
```
Numbers less than 1 should have leading zeros --> `0.5` re our coding style guide. :) I know we don't have it set up on Trade Guys, but let's try and stay consistent with it across projects.

jnschrag commented on assets/_sass/custom-viz/boeing/_base.scss:
```diff
@@ -24,14 +24,31 @@
+  }
+}
+
+.timer {
```
Be careful about prefixing/limiting the specificity of styles for the interactive. You don't want them to accidentally interfere with other items on the page.

dlousa92 commented:
Thank you for the notes! I'll make these fixes.

dlousa92 commented:
I've made the suggested changes. Two things I noticed

1. I couldn't find anywhere where the .timer class was being used so I just removed it.
2. I rearranged base.scss to apply styles specifically to elements in the interactive, but I wasn't sure about the .wrapper class styles. I moved them into what I think is more applicable and specific, but let me know if that's not quite right.

jnschrag commented:
@dlousa92 The `.wrapper` styles can be left as they were. We talked about what that is for yesterday.

Re the `.gitignore`, the reason why it isn't working is because the `.map` file has already been stored in the repo. So you'll need to create a new commit that removes it from the remote repo. You can do that like this:
```
git rm --cached /assets/js/**/*.js.map
git commit -m "remove .map from repo"
```

dlousa92 commented:
ahhh, got it thank you.
